stdlib/experimental/promql/promql.flux

```diff
@@ -0,0 +1,20 @@
+package promql
```

Comment:
Can we place this method in the `internal/promql` package?

Reply:
Yeah, let me go ahead and update that.

Reply:
@jsternberg I moved the files around so that they are in the correct folders now.